packages/optimizers/terser/src/TerserOptimizer.js

```diff
@@ -3,30 +3,45 @@
 import nullthrows from 'nullthrows';
 import {minify} from 'terser';
 import {Optimizer} from '@parcel/plugin';
-import {blobToString, loadConfig} from '@parcel/utils';
+import {blobToString} from '@parcel/utils';
 import SourceMap from '@parcel/source-map';
 import ThrowableDiagnostic, {escapeMarkdown} from '@parcel/diagnostic';
 
 import path from 'path';
 
 export default (new Optimizer({
-  async optimize({contents, map, bundle, options, getSourceMapReference}) {
+  async loadConfig({config, options}) {
+    let userConfig = await config.getConfigFrom(
+      path.join(options.entryRoot, 'index'),
+      ['.terserrc', '.uglifyrc', '.uglifyrc.js', '.terserrc.js'],
```

Comment:
While we're at it: uglify is being actively development now, so I think `.uglifyrc` shouldn't be used here